examples/clusters-dynamic.html

```diff
@@ -0,0 +1,16 @@
+---
+layout: example.html
+title: Dynamic clusters
+shortdesc: Clusters with zoom-to-cluster, hull view, and uncluttering of overlapping features.
+docs: >
+  <p>This example shows open data of subsidized photovoltaic installations in Vienna. Different style functions
+    are used for cluster display, single feature display, convex hull of a cluster, and an uncluttered view of
+    overlapping features. Hovering over a cluster shows its convex hull. Clicking on a cluster zooms to the
+    extent of the contained features. Clicking on a cluster consisting of features that are very close to each other reveals an uncluttered view
```

Comment:
Did you mean "unclustered" instead of "uncluttered" (above as well)?

Reply:
Not really "unclustered". It's a view for features that share the same location (within one pixel), or are still clustered at the view's `maxZoom`. How would you describe that?

<img width="688" alt="Untitled" src="https://user-images.githubusercontent.com/211514/147456950-a4facbf2-c7ee-47d4-85e7-335845a8e2c9.png">

The convex hulls are supposed to be rendered below the cluster circle.

Reply:
Ah, cool.  How about an “expanded” view?

Reply:
Good suggestion, thanks! I changed all occurrences to "expanded".